Packages/Sites/Neos.NeosIo/Resources/Private/Scss/Molecules/_BoxAside.scss

```diff
@@ -0,0 +1,29 @@
+.boxAside {
+	.main {
+
+		@media (max-width: 979px) and (min-width: 768px) {
```

Comment:
This breaks encapsulation of media queries. Are these break points rare exceptions? Otherwise, I'd suggest to put them aside the others (like `medium` or `large`).